camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -1122,12 +1254,115 @@ export default class Tasklist extends Mixins(TaskListMixin) {
     localStorage.removeItem("authToken");
     localStorage.removeItem("UserDetails");
   }
-  beforeCreate () {
+  beforeCreate() {
     localStorage.removeItem("authToken");
     localStorage.removeItem("UserDetails");
-    if (!this.$store.hasModule("serviceFlowModule")) {
+    if (!this.$store?.hasModule("serviceFlowModule")) {
       this.$store.registerModule("serviceFlowModule", serviceFlowModule);
     }
   }
 }
 </script>
+
+<style lang="scss" scoped>
```

Comment:
qn to the team, are we writing the SCSS in vue file or we are planning to move to a .scss file